srcpkgs/xbps-triggers/files/initramfs-regenerate

```diff
@@ -61,6 +61,17 @@ case "${INITRAMFS_GENERATOR:-dracut}" in
 			done
 		fi
 		;;
+	booster)
+		if command -v booster >dev/null 2>&1; then
+			echo "Regenerating initramfs with booster"
+			# Regenerate images for every kernel version with modules
+			for kdir in usr/lib/modules/*; do
+				[ -d "${kdir}/kernel" ] || continue
+				kver="${kdir##*/}"
+				booster -force -output "boot/initramfs-${kver}.img" -kernelVersion "${kver}"
```

Comment:
This line needs to be changed to `booster build --force --kernel-version "${kver}" "boot/initramfs-${kver}.img"`